### PR TITLE
CORDA-3200 Fix postgres oid/ bytea column issue

### PR DESCRIPTION
**Pull request by willhr3**
Add postgres specific migration to change from `@Lob` to `@Type(type = "corda-blob")`

### CODE REVIEW

File: node/src/main/resources/migration/node-core.changelog-v15.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            <not>
+                <sqlCheck expectedResult="bytea">
+                    SELECT DATA_TYPE FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = 'node_identities' AND COLUMN_NAME = 'identity_value'
+                </sqlCheck>
+            </not>
+        </preConditions>
+
+        <dropColumn tableName="node_identities" columnName="identity_value"/>
+        <addColumn tableName="node_identities">
+            <column name="identity_value" type="varbinary(33554432)">
```
Comment (josecoll) [inline node/src/main/resources/migration/node-core.changelog-v15.xml]: This needs adjusting to the maximum size of a stringified representation of a  `CordaX500Name`.

**Review by josecoll — CHANGES_REQUESTED** [review]
Looks good!
Just need to adjust the column size to something close to a CordaX500Name.

File: node/src/main/resources/migration/node-core.changelog-v15.xml (hunk 1 above)
Comment (josecoll) [reply node/src/main/resources/migration/node-core.changelog-v15.xml]: Apologies - my mistake.
I thought this was the new `node_identities_no_cert` table.

Comment (josecoll) [reply node/src/main/resources/migration/node-core.changelog-v15.xml]: `varbinary(33554432)` seems to have been copied form the checkpoints table (Eg. checkpoint blob size).
Do we have a closer sizing approximation to a serialized `PartyAndCertificate` ?

Comment (willhr3) [reply node/src/main/resources/migration/node-core.changelog-v15.xml]: Quick test showed ~6000 for a `PartyAndCertificate` so we've set the length to 64000 to cater for large cert chains.

**Review by josecoll — APPROVED** [review]

**Pull request merged**